### ISSUE TITLE
Deploy Concourse using Helm is not working

### ISSUE THREAD

**Issue by vinitajoshi** [helm/charts#5236]
I am trying to install Concourse using Helm on GKE cluster. I am following solution in https://cloud.google.com/solutions/continuous-integration-helm-concourse#deploy_concourse. 

My concourse.yaml -> 
concourse:
  baggageclaimDriver: overlay
  password: $PASSWORD
web:
  service:
  type: LoadBalancer

When I run command
./helm install stable/concourse --name concourse -f concourse.yaml --version 0.10.0
It throws below error - 
2018/04/24 17:28:15 warning: cannot overwrite table with non table for service (map[type:ClusterIP])
Error: render error in "concourse/templates/web-svc.yaml": template: concourse/templates/web-svc.yaml:11:18: executing "concourse/templates/web-svc.yaml" at <.V
alues.web.service....>: can't evaluate field type in type interface {}

Any idea on this issue?

helm version
Client: &version.Version{SemVer:"v2.6.2", GitCommit:"be3ae4ea91b2960be98c07e8f73754e67e87963c", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.6.2", GitCommit:"be3ae4ea91b2960be98c07e8f73754e67e87963c", GitTreeState:"clean"}

kubectl version
Client Version: version.Info{Major:"1", Minor:"8", GitVersion:"v1.8.6", GitCommit:"6260bb08c46c31eea6cb538b34a9ceb3e406689c", GitTreeState:"clean", BuildDate:"2
017-12-21T06:34:11Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"8+", GitVersion:"v1.8.8-gke.0", GitCommit:"6e5b33a290a99c067003632e0fd6be0ead48b233", GitTreeState:"clean", Build
Date:"2018-02-16T18:26:58Z", GoVersion:"go1.8.3b4", Compiler:"gc", Platform:"linux/amd64"}

**Comment by brunoban** [comment]
Is your concourse.yaml formatted exactly like that?

I ask because format is very important in yaml files. From the error it seems it's trying to overwrite a table variable with a non table variable, which might be the case if you are having a flat yaml file like that. Make sure you have this exact hierarchy:

```
concourse:
  password: $PASSWORD
  baggageclaimDriver: overlay
web:
  service:
    type: LoadBalancer
```

By the way tho, the current format for the latest `values.yaml` is different from the one that is on this tutorial. If you are deploying the latest concourse chart, it would look like this:

```
concourse:
  baggageclaimDriver: overlay
web:
  service:
    type: LoadBalancer
secrets:
  basicAuthPassword: $PASSWORD
```

I feel that it's always useful to just copy the current `values.yaml` from the chart repo and do the modifications directly there. It helps in these sort of situations where the format changes for any reason.

**Comment by william-tran** [comment]
There have been breaking changes to the chart this year, but the tutorial does specify a fixed version for the chart: ` --version 0.10.0`. It's likely a yaml indentation problem in the input.

**Comment by brunoban** [comment]
Well pointed out @william-tran , I didn't pay enough attention to the tutorial 😄
